src/cvode/cvode_ls.c

```diff
@@ -914,17 +929,19 @@ char *CVodeGetLinReturnFlagName(long int flag)
   The return value is the same as the value returned by jtimes --
   0 if successful, nonzero otherwise.
   -----------------------------------------------------------------*/
-int cvLsATimes(void *cvode_mem, N_Vector v, N_Vector z)
+SUNLsStatus cvLsATimes(void *cvode_mem, N_Vector v, N_Vector z)
```

Comment:
This routine has multiple inconsistencies in `int` vs `SUNLsStatus`, e.g., it internally assumes that `0` and `CVLS_SUCCESS` both equal `SUNLS_SUCCESS` and that a `CVLsJacTimesVecFn` returns values with `SUNLsStatus` type.